chart/templates/deployment.yaml

```diff
@@ -48,7 +48,7 @@ spec:
           - name: env
             value: 'production'
           - name: EVENTBROKER
-            value: 'http://localhost:8081/event'
+            value: 'nats://keptn-nats'
```

Comment:
Is the `EVENTBROKER` environment variable actually still used? The new code doesn't seem to contain any call to `os.Getenv` anymore

Reply:
Nice catch. I removed it now.